src/test/java/com/google/jenkins/plugins/computeengine/integration/CleanLostNodesWorkIT.java

```diff
@@ -0,0 +1,210 @@
+package com.google.jenkins.plugins.computeengine.integration;
+
+import static com.google.jenkins.plugins.computeengine.integration.ITUtil.LABEL;
+import static com.google.jenkins.plugins.computeengine.integration.ITUtil.NULL_TEMPLATE;
+import static com.google.jenkins.plugins.computeengine.integration.ITUtil.NUM_EXECUTORS;
+import static com.google.jenkins.plugins.computeengine.integration.ITUtil.getLabel;
+import static com.google.jenkins.plugins.computeengine.integration.ITUtil.initCloud;
+import static com.google.jenkins.plugins.computeengine.integration.ITUtil.initCredentials;
+import static com.google.jenkins.plugins.computeengine.integration.ITUtil.instanceConfigurationBuilder;
+import static com.google.jenkins.plugins.computeengine.integration.ITUtil.teardownResources;
+import static org.awaitility.Awaitility.await;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+
+import com.google.common.collect.ImmutableList;
+import com.google.jenkins.plugins.computeengine.CleanLostNodesWork;
+import com.google.jenkins.plugins.computeengine.ComputeEngineCloud;
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+import java.util.Stack;
+import java.util.concurrent.TimeUnit;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import org.jenkinsci.plugins.workflow.cps.CpsFlowDefinition;
+import org.jenkinsci.plugins.workflow.job.WorkflowJob;
```

Comment:
Missing dep? Does not compile.

Reply:
actually I apply the PR (https://github.com/jenkinsci/google-compute-engine-plugin/pull/495) with `git merge --squash` before working on the tests, and then `git revert` that commit.

Reply:
all good as the #495 already merged. (ty for help there too)